src/frontend/lexer.mll

```diff
@@ -139,6 +139,8 @@ rule token = parse
                                 Parser.CHOLESKYFACTORCOV }
   | "corr_matrix"             { lexer_logger "corr_matrix" ; Parser.CORRMATRIX }
   | "cov_matrix"              { lexer_logger "cov_matrix" ; Parser.COVMATRIX }
+  | "stochastic_column_matrix"              { lexer_logger "stochastic_column_matrix" ; Parser.STOCHASTICCOLUMNMATRIX }
+  | "stochastic_row_matrix"              { lexer_logger "stochastic_row_matrix" ; Parser.STOCHASTICROWMATRIX }
```

Comment:
General question: would it be better to make these e.g. `row_stochastic_matrix`? I'm thinking particularly if we ever support a doubly stochastic type, it would have to have an awkward name if we wanted the word stochastic to go first.

Reply:
We could also consider `row_stochastic` (drop the `_matrix`) if we wanted a shorter name -- probably worth asking @bob-carpenter